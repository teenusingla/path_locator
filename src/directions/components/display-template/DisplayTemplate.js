import React from "react";
import "./DisplayTemplate.scss";

/**
 * @name DisplayTemplate
 * @type {Component}
 * @description Display one way data
 * @param {{label,value}} Object Props to stateless component
 * @returns {JSX}
 */

const DisplayTemplate = ({ label, value }) => (
  <div className="display-info">
    <div className="label">{label}: </div>
    <div className="value"> {value}</div>
  </div>
);

export { DisplayTemplate };
