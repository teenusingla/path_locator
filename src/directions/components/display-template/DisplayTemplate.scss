.display-info {
  display: flex;
  font-size: 1em;

  &:not(:last-child) {
    margin-right: 16px;
  }
}

.display-info .label {
  font-weight: 500;
}

.display-info .value {
  margin-left: 8px;
  font-weight: bold;
}
