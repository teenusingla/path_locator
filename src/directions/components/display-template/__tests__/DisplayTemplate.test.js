import React from "react";
import renderer from "react-test-renderer";
import DisplayTemplate from "../DisplayTemplate";

describe("<DisplayTemplate /> component", () => {
  it("should match the snapshot", () => {
    const tree = renderer.create(<DisplayTemplate />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
