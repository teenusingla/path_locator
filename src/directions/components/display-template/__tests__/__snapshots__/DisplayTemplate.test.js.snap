// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayTemplate /> component should match the snapshot 1`] = `
<div
  className="display-info"
>
  <div
    className="label"
  >
    : 
  </div>
  <div
    className="value"
  >
     
  </div>
</div>
`;
